Gemfile.lock

```diff
@@ -294,8 +294,13 @@ GEM
       mime-types
       mimemagic (~> 0.3.0)
       terrapin (~> 0.6.0)
+<<<<<<< HEAD
```

Comment:
Oops it looks like there was a merge conflict that needs resolving here. Can I help out?